### DISCUSSION TITLE
Argo-CD in internal cluster to manage EKS-cluster applications

### DISCUSSION THREAD

**Discussion by b3n3w** [argoproj/argo-cd#12081] [Q&A]
Hi there, 

we`re currently experimenting a little with argo-cd as we want to go more into gitops direction. Our current problem is, that all our repositories (git and helm) are only reachable internally and not from the clusters (company compliance restrict it).
Therefore we thought, that we could run argo-cd in an selfhosted cluster (such as minikube or openshift), which can access our private git and then connect argo-cd with our EKS clusters for deployment. 

Do you have any ideas or doubts about this ? 
Do you think minikube could be the right choice for something like that ? 

Thanks !

**Comment by yebolenko** [comment]
Hi 
You can mirror your private git repo to AWS Code Commit repo
ArgoCD will connect to AWS Code Commit repo

**Comment by yebolenko** [comment]
if it's interesting for you respond here and I will add all required details
